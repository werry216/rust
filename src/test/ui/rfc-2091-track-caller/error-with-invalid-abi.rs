#![feature(track_caller)]

#[track_caller]
extern "C" fn f() {}
//~^^ ERROR rust ABI is required to use `#[track_caller]`

fn main() {}
