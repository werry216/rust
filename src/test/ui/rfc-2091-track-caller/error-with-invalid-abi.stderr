error[E0737]: rust ABI is required to use `#[track_caller]`
  --> $DIR/error-with-invalid-abi.rs:3:1
   |
LL | #[track_caller]
   | ^^^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0737`.
